src/db/options.rs

```diff
@@ -103,6 +103,9 @@ pub struct CreateCollectionOptions {
         serialize_with = "bson_util::serialize_duration_option_as_int_secs"
     )]
     pub expire_after_seconds: Option<Duration>,
+
+    /// Options for supporting change stream pre- and post-images.
+    pub change_stream_pre_and_post_images: Option<ChangeStreamPreAndPostImages>,
```

Comment:
are we actually testing / exercising this option anywhere? it doesn't look like it from the spec tests, which always use collMod to configure the option after the fact.

Reply:
Nope, it's not covered by the spec tests :(

Reply:
can we add a test of our own then?

Reply:
Done.